src/js/components/user/customDomainModal.tsx

```diff
@@ -53,30 +76,48 @@ const CustomDomainModal = ({ isOpen, toggleModal }: Props) => {
         />,
       ]}
     >
-      <form className="slds-p-around_large" onSubmit={handleSubmit}>
-        <div className="slds-form-element__help slds-p-bottom_small">
-          {i18n.t(
-            'To go to your company’s login page, enter the custom domain name.',
-          )}
-        </div>
-        <Input
-          id="login-custom-domain"
-          label={i18n.t('Custom Domain')}
-          value={url}
-          onChange={handleChange}
-          aria-describedby="login-custom-domain-help"
-        >
-          <div
-            id="login-custom-domain-help"
-            className="slds-form-element__help slds-truncate slds-p-top_small"
-            data-testid="custom-domain"
-          >
-            https://
-            {url.trim() ? url.trim() : <em>domain</em>}
-            .my.salesforce.com
+      {org === 'custom' && (
+        <form className="slds-p-around_large" onSubmit={handleSubmit}>
+          <div className="slds-form-element__help slds-p-bottom_small">
+            {i18n.t(
+              'To go to your company’s login page, enter the custom domain name.',
+            )}
           </div>
-        </Input>
-      </form>
+          <Input
+            id="login-custom-domain"
+            label={i18n.t('Custom Domain')}
+            value={url}
+            onChange={handleChange}
+            aria-describedby="login-custom-domain-help"
+          >
+            <div
+              id="login-custom-domain-help"
+              className="slds-form-element__help slds-truncate slds-p-top_small"
+              data-testid="custom-domain"
+            >
+              https://
+              {url.trim() ? url.trim() : <em>domain</em>}
+              .my.salesforce.com
+            </div>
+          </Input>
+        </form>
+      )}
+      {org === 'dev' && (
+        <div className="slds-p-around_large">
+          <Button
+            label={i18n.t('Log into Developer Edition Org with DevHub')}
+            variant="brand"
+            style={{ width: `100%` }}
+            onClick={handleSubmit}
+          />
+          <Button
+            label={i18n.t('Log into Custom Org with DevHub')}
+            variant="outline-brand"
+            style={{ width: `100%` }}
+            onClick={() => setOrg('custom')}
+          />
```

Comment:
These button labels don't make sense. The login.salesforce.com endpoint will work for both production and Developer Edition orgs, and those same orgs can alternatively be accessed using the custom domain login endpoint.